packages/wallets/backpack/src/adapter.ts

```diff
@@ -173,23 +167,22 @@ export class BackpackWalletAdapter extends BaseMessageSignerWalletAdapter {
 
     async sendTransaction(
         transaction: Transaction,
-        connection: Connection,
+        _connection: Connection,
```

Comment:
Am I correct in thinking we always want to defer to Backpack for the connection?

Reply:
Ideally, yes. Though in the event users want to provide their own connection, we should let them.